src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java

```diff
@@ -38,6 +39,14 @@ public TankDrivetrain(MotorController leftController, MotorController rightContr
         this(getClassName(DEFAULT_NAMESPACE_NAME), leftController, rightController);
     }
 
+    public TankDrivetrain(MotorController leftMaster, MotorController leftSlave, MotorController rightMaster, MotorController rightSlave) {
+        this(getClassName(DEFAULT_NAMESPACE_NAME), new MotorControllerGroup(leftMaster, leftSlave), new MotorControllerGroup(rightMaster, rightSlave));
```

Comment:
instead of creating MotorControllerGroups, use the constructor (String, MotorController, MotorController, MotorController, MotorController)